deployment/remediations-update-consumer-clowdapp.yaml

```diff
@@ -119,6 +121,8 @@ parameters:
   value: 'false'
 - name: VULNERABILITY_TOPIC_ENABLED
   value: 'true'
+- name: PLAYBOOK_DISPATCHER_TOPIC_ENABLED
+  value: 'true'
```

Comment:
I thinkkk it should be set to true here and false in `deployment/remediations-consumer-clowdapp.yaml` 🤔